src/source/PeerConnection/PeerConnection.h

```diff
@@ -21,9 +21,9 @@ extern "C" {
 // A non-comprehensive list of valid JSON characters
 #define VALID_CHAR_SET_FOR_JSON         "0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz+/"
 
-#define ICE_CANDIDATE_JSON_TEMPLATE     (PCHAR) "{\"candidate\":\"candidate:%s\",\"sdpMid\":\"0\",\"sdpMLineIndex\":0}"
-
-#define MAX_ICE_CANDIDATE_JSON_LEN      (MAX_SDP_ATTRIBUTE_VALUE_LENGTH + STRLEN(ICE_CANDIDATE_JSON_TEMPLATE) + 1)
+#define ICE_CANDIDATE_JSON_TEMPLATE     (PCHAR) "{\"candidate\":\"candidate:%s\",\"sdpMid\":\"0\",\"sdpMLineIndex\":0}" // SIZE 59
```

Comment:
Please don't use line comments after the macro definitions. Move up on top of the macro

Reply:
If I use the original code here windows give me this error:
```
Error (active)	E0059	function call is not allowed in a constant expression	kvsWebrtcClient	C:\Exp\aws\src\source\PeerConnection\PeerConnection.c	317	
```
That the reason why I hardcode the 59 base on the value of STRLEN